tests/manual_test_tethfinex.py

```diff
@@ -21,62 +21,97 @@
 
 from pyexchange.tethfinex import TEthfinexToken, TEthfinexApi
 from pymaker import Address, Wad
-from pymaker.keys import register_key_file
-from pymaker.zrx import ZrxExchange
-
+from pymaker.keys import register_key_file, register_private_key
+from pymaker.zrxv2 import ZrxExchangeV2, Asset
+from pymaker.token import ERC20Token
 
+# Infura and own wallet settings
 logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s', level=logging.INFO)
+WEB3_INFURA_API_KEY = "infura project id here"
+INFURA_URL = "https://mainnet.infura.io/v3/" + WEB3_INFURA_API_KEY
+WALLET_ADDRESS = "eth wallet address here"
+WALLET_PRIVATE_KEY = "pkey of eth wallet address here"
+
+# setting Infura provider and import external pkey
+infura_provider = HTTPProvider(INFURA_URL)
+web3 = Web3(infura_provider)
+register_private_key(web3, WALLET_PRIVATE_KEY)
```

Comment:
just a note - we're using register_key_file rather then working with private key